docs/modules/development/pages/rest/config_management.adoc

```diff
@@ -0,0 +1,65 @@
+= Config Management
+
+Use the Config Management REST API to manage configurations for OpenNMS.
+Configurations modified will take effect instantly.
+
+== GETs (Reading Data)
+
+[options="header", cols="5,10"]
+|===
```

Comment:
Should remove the `/rest` for each of the endpoints, for consistency with the other REST doc pages